app/response_processor.py

```diff
@@ -1,16 +1,25 @@
 from json import dumps
+import logging
 import os
 
 from requests.packages.urllib3.exceptions import MaxRetryError
+from sdc.rabbit.exceptions import BadMessageError, PublishMessageError, RetryableError
+from structlog import wrap_logger
 
 from app import settings
 from app.private_publisher import PrivatePublisher
 from app.settings import session
-from app.helpers.exceptions import DecryptError, BadMessageError, RetryableError
+
+logger = wrap_logger(logging.getLogger(__name__))
 
 
 class ResponseProcessor:
 
+    @classmethod
```

Comment:
This seems obtuse, considering we just need to supply a callable - what is the benefit over instantiating an instance of this class, and accessing the process method via it?